app/src/layers/components/LayerPanel.jsx

```diff
@@ -72,17 +72,17 @@ class LayerPanel extends Component {
           </ul >
         </ExpandItem >
         <AccordionHeader
-          menuName={'Map Layers'}
```

Comment:
Are we sure the client asked us to rename "Map Layers" and not "Fishing Layers"? this doesn't make much sense IMO

Reply:
I understood they wanted us to change the name because this layers will not be only fishing layers but also will contain other information like encounters

Reply:
Just confirmed with Enrique, to be sure, and it's the other group. In any case, please remove this change from this PR, I will handle it separately